src/common/transformations/tests/transpose_sinking/ts_common_test.cpp

```diff
@@ -114,13 +114,30 @@ class PadFactory : public IFactory {
 public:
     explicit PadFactory(const std::string& type_name) : IFactory(type_name) {}
     NodePtr create(const OutputVector& parent_nodes) const override {
-        return std::make_shared<Pad>(parent_nodes[0], parent_nodes[1], parent_nodes[2], ov::op::PadMode::CONSTANT);
+        return std::make_shared<ov::op::v1::Pad>(parent_nodes[0],
+                                                 parent_nodes[1],
+                                                 parent_nodes[2],
+                                                 ov::op::PadMode::CONSTANT);
```

Comment:
There are `using namespace std;` and `using namespace ov;` at the front of the file (applies elsewhere `std::` or `ov::` are used)

Reply:
let's change this in the next PR